datafusion/common/src/config.rs

```diff
@@ -1665,6 +1670,14 @@ impl CsvOptions {
         self
     }
 
+    /// Set true to ensure that newlines in (quoted) values are supported.
```

Comment:
```suggestion
    /// Set true to support newlines in (quoted) values.
```

Reply:
The phrasing was a bit awkward initially as I was thinking of the flag as "ensuring support", rather than directly "supporting", since newlines in values are already supported if the file is below the `datafusion.optimizer.repartition_file_min_size`.

I'm not sure if it's worth conveying that detail through these docs, or else to document this as providing support and treat the fact that newlines in values will "just work" for smaller files as an implementation detail that might change.

Reply:
I've expanded and normalised the documentation. I've gone for documenting the flag as enabling support for newlines in values, with the fact that newlines in values might work without it left as an implementation-defined detail.